packages/libs/lambda-at-edge/src/images/imageOptimizer.ts

```diff
@@ -0,0 +1,400 @@
+/**
```

Comment:
Did you consider using next's exported image-optimiser instead of copying it over? i.e.
```import imageOptimizer from "next/dist/next-server/server/image-optimizer"```

Reply:
Oh I didn't but I think it doesn't exactly work because it tries to write to the local directory and also it has some extra code that isn't needed

Reply:
Fair enough

Reply:
And also it tries to read images from local directory but we have it on s3 instead